index.bs

```diff
@@ -405,6 +424,8 @@ When this method is invoked on an {{XRFrame}} |frame|, the user agent MUST run t
 
 </div>
 
+NOTE: if any of the spaces belonging to the same {{XRHand}} return <code>null</null> when [=hands/populate the pose|populating the pose=], all the spaces of that {{XRHand}} must also return <code>null</null> when [=hands/populate the pose|populating the pose=]
```

Comment:
It' s not clear to me how this note significantly differs from the one on line 392? They're both saying the same thing, just using pointing at different mechanisms to say it.

Reply:
I did that to clarify each algorithm.